aws/README.md

```diff
@@ -104,12 +104,17 @@ tag_instance.ps1
 ```
 
 
-## update_other_tag.py
+## update_costcenter_tag.py
 
-Set the value of CostCenterOther for a specified EC2 instance, including related resources in the same cloudformation stack
-and/or service catalog product. This is useful for fixing typos when launching a service catalog product or for migrating a
-cloudformation stack from one cost center to another. The CostCenterOther tag will only be updated if the CostCenter tag is
-"Other / 000001" and the new value for CostCenterOther is valid.
+Set the value of CostCenter and/or CostCenterOther for a specified resource,
+including related resources in the same cloudformation stack and/or service
+catalog product. The specified resource can be either an EC2 instance-id,
+or an arbitrary ARN. The CostCenterOther tag will only be updated if the
+CostCenter tag on the specified resource starts with "Other", otherwise it will
```

Comment:
not clear to me.  what will be removed if present?

Reply:
the CostCenterOther tag, I've edited this for clarity

Reply:
sorry, it's still unclear to me what's gonna happen.  Do you mean that the CostCenterOther tag will be removed if `CostCenter != other`?  if so what if a resource contains both tags, CostCenter and CostCenterOther where  `CostCenter != other`?

Reply:
If a resource contains both CostCenter and CostCenterOther, and CostCenter is not Other, then that's an error, it should never happen. and if it does happen, I'm not sure if cost explorer would duplicate the cost in both cost centers or if it would choose one of the two (and it might surprise the owner which one it chooses). but either way, it's important to make sure that CostCenterOther is not present when CostCenter is not Other for the cost to be tracked accurately.

Reply:
> If a resource contains both CostCenter and CostCenterOther, and CostCenter is not Other, then that's an error, it should never happen

Its very possible for that to happen and it will not be an error at all when it happens in the service catalog. I would double check existing service catalog resources to make sure that it hasn't happened.

I could image users may want to use both tags in an attempt to split costs between two cost centers.

Reply:
I know that we don't have a technical way of preventing it from happening, but it's still a user error when it happens, and we can use this script to fix any cases that we find. also, I guess it makes sense to add a check for mismatched tags to the check looking for missing CostCenterOther tags.

Reply:
I've created Sage-Bionetworks-IT/lambda-finops-email-totals#21 to also check for unexpected CostCenterOther tags.

Reply:
> I've created [Sage-Bionetworks-IT/lambda-finops-email-totals#21](https://github.com/Sage-Bionetworks-IT/lambda-finops-email-totals/pull/21) to also check for unexpected CostCenterOther tags.

i see the relationship however i don't see how that PR addresses my original confusion around what happens?  Would you be able to add some more docs here to clarify to the user that runs this script what the intentions are?

Reply:
yeah, I've expanded the README description to include the use-case of needing to enforce a tag constraint to help explain how the tags are updated